src/components/Heroes/index.jsx

```diff
@@ -100,7 +100,13 @@ class RequestLayer extends React.Component {
             icon=""
             twoLine
           />
-          <Table data={data} columns={_columns} loading={loading} />
+          <div style={{position:"relative"}}>
+            <Table data={data} columns={_columns} loading={loading} />
+            <Table data={data} columns={[_columns[0]]} loading={loading} fixedStyle={{
```

Comment:
Well, I have a suggestion about `column`which is more  flexible. 
maybe you can add a prop  `fixed: right(or left)` in the column items, then you can fix some of columns right or left.

Reply:
I like this suggestion. Would the idea be that if multiple items had a prop of "fixed:right" that they would stack and be fixed to the right?

Reply:
I think you could create a file of column tools, then you can filter some of columns use `Array.filter` or something else.
and if the filter column exists, such as 
```
const rightFixedColumns = columnTools.getRight(columns);
const leftFixedColumns = columnTools.getLeft(columns);
```
after that, use your code to create two fixed table above the original table. located right and left.

Sorry, I don't know whether you understand what I said. My English is not good.

Reply:
No worries, that explanation is perfect. I'll tackle this as soon as I work through the kinks of the fixed header.